packs/mmonit/sensors/events_sensor.yaml

```diff
@@ -0,0 +1,22 @@
+---
+  class_name: "MmonitEventsSensor"
+  entry_point: "events_sensor.py"
+  description: "Sensor which monitors mmonit for events"
+  poll_interval: 10
+  trigger_types:
+    -
+      name: "new_event"
+      description: "Trigger which indicates that a new event has arrived at mmonit"
+      payload_schema:
+        type: "object"
+        properties:
+          host:
+            type: "string"
+          id:
+            type: "integer"
+          event:
```

Comment:
Well, there is quite a few values possible so that why I set up only the 3 that looked more importatn for filtering and left the rest in the whole object. 

Ill have a look tonight to see how many there are.